packages/protocol/src/re-exports.ts

```diff
@@ -13,7 +13,45 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-export { Viewport } from 'sprotty-protocol/lib/model';
+export {
+    Alignable,
+    EdgeLayoutable,
+    EdgePlacement,
+    EdgeSide,
+    Expandable,
+    Fadeable,
+    SButton as GButtonSchema,
+    SCompartment as GCompartmentSchema,
+    SEdge as GEdgeSchema,
+    ForeignObjectElement as GForeignObjectElementSchema,
+    SGraph as GGraphSchema,
+    HtmlRoot as GHtmlRootSchema,
+    SIssue as GIssue,
+    SIssueMarker as GIssueMarkerSchema,
+    SIssueSeverity as GIssueSeverity,
+    SLabel as GLabelSchema,
+    SModelElement as GModelElementSchema,
+    SModelRoot as GModelRootSchema,
+    SNode as GNodeSchema,
+    SPort as GPortSchema,
+    PreRenderedElement as GPreRenderedElementSchema,
+    SShapeElement as GShapeElementSchema,
+    ShapedPreRenderedElement as GShapedPreRenderedElementSchema,
+    ViewportRootElement as GViewportRootElementSchema,
+    HAlignment,
+    Hoverable,
+    LayoutKind,
+    Locateable,
+    ModelLayoutOptions,
+    Projectable,
+    Scrollable,
+    Selectable,
+    VAlignment,
+    Viewport,
+    Zoomable,
+    isScrollable,
+    isZoomable
+} from 'sprotty-protocol/lib/model';
```

Comment:
Are we missing some things from this commit: https://github.com/eclipse-sprotty/sprotty/commit/3a3a0a0ec9a6d46cbafc1d632751d922fd67a634#diff-e79f06c7d954f3bab90fefa6a847d39bacb660d538d281b0703a58fd6c092b8d?

I'm just wondering because sprotty seems to have made a split between the external `BoundsAware` (position and size property) and the internal `InternalBoundsAware` (with just a bounds property). Similar "issues" with `LayoutableChild` or `ModelLayoutOptions`. In the client package we can now choose to import from `@eclipse-glsp/sprotty` or `sprotty-protocol` and the latter results in a warning that we should use the former but they are not actually interchangeable. I wonder how we should best proceed with this but we definitely should also export the sprotty-protocol interfaces as well somehow, what do you think?

Reply:
In GLSP we only use the client side model of sprotty (what sprotty calls internal model). If there are differences between sprotty's interfaces in the internal model and their transfer model (SModel resp. GModelSchema in GLSP) we have to make sure that we always reexport the corresponding internal interface (since that is the one we are actually using).

The name clash is unfortunate but their is no easy way resolve this.